nilearn/plotting/html_surface.py

```diff
@@ -192,6 +192,11 @@ def view_img_on_surf(stat_map_img, surf_mesh='fsaverage5',
     title_fontsize : int, optional
         Fontsize of the title. Default=25.
 
+    vol_to_surf_kwargs : dict, optional
+        Dictionary of keyword arguments that are passed on to
+        :func:`nilearn.surface.vol_to_surf` when extracting a surface from
+        the input image. See the function documentation for details.
```

Comment:
> In my last push I've added an entry to the view_img_on_surf docstring and embedded a link to the documentation for vol_to_surf. I think this would avoid unnecessary duplication of documentation.

thanks, I agree with you about avoiding too much duplication but still if someone gets weird results when plotting an atlas and is not used to the vol_to_surf function they might not think to look in there.
So maybe we could repeat here what is the appropriate parameter when plotting an atlas for example:

this parameter is especially useful when plotting an atlas; in that case to avoid averaging or interpolating labels from neighboring voxels pass vol_to_surf_kwargs = {"n_samples": 1, "interpolation": "nearest"}

Reply:
Thank you - I've added a bit more info in the docstring and linked to the relevant example too.